databases/week2/Part3.sql

```diff
@@ -0,0 +1,32 @@
+use hyf_lesson2;
+
+#All the tasks assigned to users whose email ends in @spotify.com
+SELECT task.*
```

Comment:
It would be nice to see the user email here as well, to make sure that it ends in @spotify.com.

Reply:
Oh, yes. Good point.